pyproject.toml

```diff
@@ -23,8 +23,9 @@ dependencies = [
   "durationpy",
   "flask~=2.2",
   "prometheus_flask_exporter~=0.22",
+  "pyyaml~=6.0",
+  "timeago~=1.0",
```

Comment:
[praise] cool! didn't know this library. I always wanted to try out Arrow but never had the chance. Did you consider using that one as well?

Reply:
Never heard of Arrow. Looks cool.